package.json

```diff
@@ -29,7 +29,7 @@
         "build:dist:es": "tsc --project ./tsconfig.es.json",
         "build:dist:cjs": "tsc --project ./tsconfig.cjs.json",
         "build:assets": "npm-run-all -p build:dist:* build:webpack:prod",
-        "build": "npm-run-all -s build:schemas build:assets",
+        "build": "npm-run-all -s build:schemas update:version build:assets",
```

Comment:
suggestion: If we run any of the targets independently, like `npm run build:dist:es`, then the correct version may not be used.

This is fine for the deployment script in this repository, because it runs `build`, but there is are scenarios where this step could be missed. For example, a third party runs their own web client build, but runs only the subset they need: `build:schemas && build:dist:es`.

One way to help mitigate this would be to omit `version.json` from version control (by adding it to `.gitignore`). This way, like the schemas, `build:version` would need to run before the other build steps, helping ensure the file is up to date.

Reply:
Added to `.gitignore`